travis-emcc.sh

```diff
@@ -0,0 +1,4 @@
+set -e
```

Comment:
Do you mind moving this file to the `scripts` directory? That's where the rest of the Travis scripts are currently.

Reply:
Sure, done.